index.js

```diff
@@ -88,27 +104,54 @@ class CSVPlugin {
     return this.context.json.expand(data)
   }
 
+  loadExportTemplates(data) {
+    const state = this.context.window.store?.getState()
+    const inferredItemTemplate = data['@graph']?.[0]?.template
+    const itemTemplate = loadTemplate(
+      state,
+      this.options.itemTemplate,
+      inferredItemTemplate)
+
+    if (!itemTemplate)
+      // eslint-disable-next-line max-len
+      throw new Error(`Failed to find specified item template "${this.options.itemTemplate}" or fallback "${data['@graph']?.[0]?.template}". Please install one of these templates, or select a different template and try again.`)
+
+
+    const inferredPhotoTemplate = data['@graph']?.[0]?.photo?.[0]?.template
```

Comment:
Do we want to handle the case where the first item has no photo (but there are photos later on)? (Probably not worth the extra effort)

Reply:
is that a common situation in projects? I think I'd be tempted to not handle this case unless users run into it and it causes problems.

Reply:
Yes, items without photos will be very uncommon. Even if you have such items, I think it would be very unlikely that you'd export a mixed selection of items with and without photos -- specifically, it makes little sense to configure the plugin to export photo metadata and then use it to export items without photos. Let's punt it.